api/jobs.yaml

```diff
@@ -341,15 +268,26 @@ definitions:
           The continuation token, which is used to page through large result
           sets. To get the next page of results, set this parameter to the value
           of `nextPageToken` from the previous response.
-      parentId:
+      extendedQueryFields:
+        $ref: '#/definitions/ExtendedQueryFields'
+  ExtendedQueryFields:
+    description: >
+      Extended fields to query by which may not be supported by all backends.
+    properties:
+      projectId:
         type: string
         description: >
-          Returns only jobs with the given parent ID. This parameter may be
-          unsupported for some API implementations or required for others
-          depending on whether there exists a logical noun above a job
-          in the resource hierarchy (for example, a cloud project). If the
-          presence of this parameter is incompatible with the server, it may
-          return a 400 HTTP status.
+          Returns only jobs with the specified project ID. If specified by the
```

Comment:
nit: I'd probably say "belonging to the specified  project". And at the end "; all jobs have exactly one parent project id."